bears/general/IndentationBear.py

```diff
@@ -47,7 +48,8 @@ def run(self,
             No. of spaces to insert for indentation.
             Only Applicable if use_spaces is False.
         """
```

Comment:
there's already an issue in coala-bears regarding this IIRC i'll assign him to that :P